pkg/node/sendoperation/deploySC/deploySC.go

```diff
@@ -0,0 +1,50 @@
+package deploysc
+
+import (
+	"io"
+
+	"fyne.io/fyne/v2"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/thyra/api/swagger/server/models"
+	"github.com/massalabs/thyra/api/swagger/server/restapi/operations"
+	"github.com/massalabs/thyra/int/api/websites"
+	"github.com/massalabs/thyra/pkg/node"
+	"github.com/massalabs/thyra/pkg/onchain"
+)
+
+func CreateCmdDeploySCHandler(
+	app *fyne.App,
+) func(params operations.CmdDeploySCParams) middleware.Responder {
+	return func(params operations.CmdDeploySCParams) middleware.Responder {
+		return cmdDeploySCHandler(params, app)
+	}
+}
+
+func cmdDeploySCHandler(params operations.CmdDeploySCParams, app *fyne.App) middleware.Responder {
+	client := node.NewDefaultClient()
+
+	wallet, _ := websites.LoadAndUnprotectWallet(params.Nickname, app)
```

Comment:
This line feels weird. Why deploying a SC would need tol use a website package. I think the package are not well organized. Furthermore, I have the feeling that you should use the plugin wallet instead of the local wallet.

Reply:
Is plugin wallet installed with Thyra and Functionnal ? 

I think when @SebastienValla worked on this Plugin wallet was not ready.
And from my side i didn't know it was ready to implement

Reply:
https://github.com/massalabs/thyra/pull/442/files#r1108797984